### PR TITLE
Gracefully stop on SIGTERM.

### PR DESCRIPTION
**Pull request by merlin-northern**
ChangeLog:none
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

**Review by oleorhagen — APPROVED** [review]
Makes sense to me!

Could we just add a log message here also?

**Comment by oleorhagen** [comment]
Maybe a small changelog be nice also?

File: cli/commands.go [hunk 1]
```diff
@@ -238,6 +239,8 @@ func runDaemon(d *app.MenderDaemon) error {
 			} else if s == syscall.SIGUSR2 {
 				log.Debug("SIGUSR2 signal received.")
 				d.ForceToState <- app.States.InventoryUpdate
+			} else if s == syscall.SIGTERM {
+				d.StopDaemon()
```
Comment (kacf) [inline cli/commands.go:243]: Have you tested this? It doesn't look like this breaks out of any sleep, it just sets a flag.

Comment (oleorhagen) [reply cli/commands.go:243]: It breaks out of sleep on the following line, no?

Comment (kacf) [reply cli/commands.go:243]: Oh, lol! You're right. The codecov box hid it from me!

**Comment by merlin-northern** [comment]
> Maybe a small changelog be nice also?

added!

**Comment by merlin-northern** [comment]
> Makes sense to me!
> 
> Could we just add a log message here also?

added!

**Comment by merlin-northern** [comment]
merging as the pipeline [passed](https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/190240469)

**Pull request merged**